src/application-development/alloc.md

```diff
@@ -0,0 +1,70 @@
+# Alloc
+
+In a `no_std` environment, the [`alloc`][alloc] crate is available as an option for heap allocation. It can be useful when working with crates that require alloc or when using dynamic collections like `Vec`.
```

Comment:
```suggestion
In a `no_std` environment, the [alloc] crate is available as an option for heap allocation. It can be useful when working with crates that require alloc or when using dynamic collections like `Vec`.
```

I don't think there is much point in formatting crate names in links as code, and you could remove some noise either this way, or by deifning the anchor as ``[`alloc`]: blabla``

Reply:
the guide says it should be monospace - https://github.com/esp-rs/book/blob/main/rust-doc-style-guide.md#using-monospace:~:text=Names%20of%20crates%2C%20traits%2C%20libraries%2C

Reply:
It also says to use monospace for architectures, which are proper nouns, which IMO is insane. So I think it needs some revising.